runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -327,32 +332,37 @@ public LayoutBuilder removeSequencerServers(Set<String> endpoints)
      * Removes the logunit endpoint from the layout.
      *
      * @param endpoint Log unit server to be removed
-     * @return Workflow manager
+     * @return Builder
      * @throws LayoutModificationException Cannot remove non-replicated logunit
      */
     public LayoutBuilder removeLogunitServer(String endpoint)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 23 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)